### ISSUE TITLE
Bug report: android-sdk does not link platform-tools binaries

### ISSUE THREAD

**Issue by julienschmidt** [caskroom/homebrew-cask#31468]
[The now removed formula for android-sdk](https://github.com/Homebrew/homebrew-core/pull/11468/files)  used to link the platform tools binaries(adb, fastboot) to `/usr/local/bin/` so that they were easily available.

[The new Cask](https://github.com/caskroom/homebrew-cask/blob/3e8fb114a654d1788805b2327a9d997e19f5b62c/Casks/android-sdk.rb) does not do that, requiring users to manually find where exactly the platform-tools binaries are installed and manually link the binaries to `/usr/local/bin/`, e.g.:
```sh
$ ln -s /usr/local/Caskroom/android-sdk/25.2.3/platform-tools/adb /usr/local/bin/adb
$ ln -s /usr/local/Caskroom/android-sdk/25.2.3/platform-tools/fastboot /usr/local/bin/fastboot
```

The behavior should remain consistent, especially since the formula gets automatically replaced by this cask.

**Comment by vitorgalvao** [comment]
https://github.com/caskroom/homebrew-cask/issues/31437

**Comment by julienschmidt** [comment]
This is not a duplicate of #31437. android-platform-tool used to be a formula that provided only the platform-tools binaries, independent of the SDK. Some users don't need the full SDK and thus prefer the slim android-platform-tool formula. However, if the SDK is installed, those binaries should be installed via the SDK as other parts of the SDK depend on it and thus might require duplicate binaries otherwise. That's why there used to be two independent formulas, which however blocked each other.

**Comment by miccal** [comment]
Downloading and expanding the file from `https://dl.google.com/android/repository/tools_r25.2.3-macosx.zip`, I cannot see a binary file for `adb` and `fastboot`, are they part of `android-platform-tool`?

**Comment by julienschmidt** [comment]
No, it is not a dependency. Look at `android-platform-tool` as an alternative distribution channel for a subset of the SDK.

The binaries are installed via the sdkmanager here: [android-sdk.rb#L27](https://github.com/caskroom/homebrew-cask/blob/3e8fb114a654d1788805b2327a9d997e19f5b62c/Casks/android-sdk.rb#L27)

**Comment by GeorgThomassen** [comment]
I can't even find the SDK after updating! 😅 it sure isn't in the /opt/ folder anymore

**Comment by julienschmidt** [comment]
@GeorgThomassen Try `/usr/local/Caskroom/android-sdk/`:

```bash
$ which sdkmanager # where in the $PATH the sdkmanager can be found
/usr/local/bin/sdkmanager
$ readlink /usr/local/bin/sdkmanager # resolve the symlink
/usr/local/Caskroom/android-sdk/25.2.3/tools/bin/sdkmanager
```

**Comment by Raphx** [comment]
Since `platform-tools` need to be installed separately, I guess it needs to be symlinked after `postflight`?

**Comment by MikeMcQuaid** [comment]
This can be closed now https://github.com/caskroom/homebrew-cask/pull/31501 is merged.

**Comment by julienschmidt** [comment]
@MikeMcQuaid #31437 is still a different issue 🙄 
This is not affected by #31501

**Comment by julienschmidt** [comment]
There is a PR for this now: #31598

**Comment by MikeMcQuaid** [comment]
> @MikeMcQuaid #31437 is still a different issue 🙄

There's no need to 🙄, that's a bit rude.

**Comment by julienschmidt** [comment]
@MikeMcQuaid no rudeness intended, sorry!

I just want a clean solution for this, like we had before.

**Comment by julienschmidt** [comment]
Fixed by #31568
